custom_reports/columns.rst

```diff
@@ -514,6 +514,16 @@ Note: "Database type" column presents the type of source column of the dimension
     +--------------------------------------+---------------------------------------+-------+-----------+-------------------+--------+--------------------------------------------------------------------------------------------------+
     |Product category (Level 5)            |product_category5                      |product|str        |not applicable     |False   |not available in Raw data API                                                                     |
     +--------------------------------------+---------------------------------------+-------+-----------+-------------------+--------+--------------------------------------------------------------------------------------------------+
+    |Product dimension 1                   |product_dimension1                     |product|str        |not applicable     |False   |                                                                                                  |
+    +--------------------------------------+---------------------------------------+-------+-----------+-------------------+--------+--------------------------------------------------------------------------------------------------+
+    |Product dimension 2                   |product_dimension2                     |product|str        |not applicable     |False   |                                                                                                  |
+    +--------------------------------------+---------------------------------------+-------+-----------+-------------------+--------+--------------------------------------------------------------------------------------------------+
+    |Product dimension 3                   |product_dimension3                     |product|str        |not applicable     |False   |                                                                                                  |
+    +--------------------------------------+---------------------------------------+-------+-----------+-------------------+--------+--------------------------------------------------------------------------------------------------+
+    |Product dimension 4                   |product_dimension4                     |product|str        |not applicable     |False   |                                                                                                  |
+    +--------------------------------------+---------------------------------------+-------+-----------+-------------------+--------+--------------------------------------------------------------------------------------------------+
+    |Product dimension 5                   |product_dimension5                     |product|str        |not applicable     |False   |                                                                                                  |
```

Comment:
hmm.. there should be 20 of them.. maybe that's some dev-override of `product_custom_dimension` setting? :thinking:

Reply:
`PRODUCT_CUSTOM_DIMENSIONS_MAX_SLOTS = 20` seems it's not always produce correct results. I will add these manually this time